products/BellHybrid/apps/common/include/common/layouts/HomeScreenLayoutClassicWithQuotes.hpp

```diff
@@ -12,16 +12,12 @@ namespace gui
       public:
         HomeScreenLayoutClassicWithQuotes(std::string name);
```

Comment:
Consider marking this constructor `explicit`, as it takes only one parameter, also passing the name via `const &` should avoid one unnecessary copying of the string.

Reply:
done